internal/cluster/cluster.go

```diff
@@ -101,8 +101,9 @@ type PrepareConfig struct {
 }
 
 type Config struct {
-	PrepareConfig    PrepareConfig
-	MonitorBatchSize int `envconfig:"CLUSTER_MONITOR_BATCH_SIZE" default:"100"`
+	PrepareConfig           PrepareConfig
+	MonitorBatchSize        int  `envconfig:"CLUSTER_MONITOR_BATCH_SIZE" default:"100"`
+	EnableSingleNodeDnsmasq bool `envconfig:"ENABLE_SINGLE_NODE_DNSMASQ" default:"false"`
```

Comment:
why sometimes the default is true and sometimes it's false?

Reply:
In code we should have false. Testing, cloud and on-prem should have true